ush/python/pygfs/jedi/jedi.py

```diff
@@ -216,18 +218,51 @@ def get_bias_dict(self, task_config: AttrDict) -> Dict[str, Any]:
                 obfile = ob['obs bias']['input file']
                 obdir = os.path.dirname(obfile)
                 basename = os.path.basename(obfile)
-                prefix = '.'.join(basename.split('.')[:-2])
-                for file in ['satbias.nc', 'satbias_cov.nc', 'tlapse.txt']:
-                    bfile = f"{prefix}.{file}"
-                    copylist.append([os.path.join(task_config.COM_ATMOS_ANALYSIS_PREV, bfile), os.path.join(obdir, bfile)])
-                    # TODO: Why is this specific to ATMOS?
+                prefix = '.'.join(basename.split('.')[:-3])
+                bfile = f"{prefix}.{bias_file}"
+                tar_file = os.path.join(obdir, bfile)
+                copylist.append([os.path.join(task_config.VarBcDir, bfile), tar_file])
+                break
 
         bias_dict = {
             'mkdir': [os.path.join(task_config.DATA, 'bc')],
             'copy': copylist
         }
+
         return bias_dict
 
+    @logit(logger)
+    def extract_tar(self, tar_file: str) -> None:
+        """Extract bias correction files from a tarball
```

Comment:
```suggestion
    @staticmethod
    @logit(logger)
    def extract_tar(tar_file: str) -> None:
        """Extract files from a tarball
```
This method does not rely on any of the object attributes, so `self` is not required, making it a `staticmethod`.
Also, the method does not know they are bias correction files.

Reply:
Thanks @aerorahul .  I saw `@staticmethod` on some stack-overflow pages but wasn't sure if this is the proper way to code this.   `@staticmethod` added and comment corrected.  Done at 821ee760.